### PR TITLE
Som devel

### PR DESCRIPTION
**Pull request by sorgom**

### CODE REVIEW

File: CLOC.md [hunk 1]
```diff
@@ -1,34 +1,34 @@
-## total lines of code: 3838
+## total lines of code: 5182
```
Comment (sombeam) [inline CLOC.md]: I'd suggest to remove this.

File: application/components/SIG/SIG_X.h [hunk 2]
```diff
@@ -98,6 +106,12 @@ class SIG_H_N : public SIG_X
 
     inline INT32 type() const { return SIG_TYPE_H_N; }
 
+private:
+    void procH0N0(INT32 speed);
```
Comment (sombeam) [inline application/components/SIG/SIG_X.h]: proc_H0_N0 ?
states are separated every where else

File: specification/doc/transitions_SIG_H.md
Comment (sombeam) [inline specification/doc/transitions_SIG_H.md:1]: remove all generated transition md files.
no one reads that.

File: testing/plans/tests_LCR_X.csv
Comment (sombeam) [inline testing/plans/tests_LCR_X.csv:1]: all test plan csv files became redundant -> remove

File: testing/tests/moduletests/LCR/LCR_01.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+    "//" : "=================================",
+    "//" : "transitions test code for LCR",
```
Comment (sombeam) [inline testing/tests/moduletests/LCR/LCR_01.json]: test code **setup** ?

File: testing/tests/moduletests/SIG/README.md [hunk 4]
```diff
@@ -6,6 +6,11 @@ test of modules SIG_(X)
 according to test plan tests_SIG_X.csv
```
Comment (sombeam) [inline testing/tests/moduletests/SIG/README.md]: test plans shall be removed.

File: testing/tests/moduletests/SIG/SIG_01.json [hunk 5]
```diff
@@ -0,0 +1,23 @@
+{
+    "//" : "=================================",
+    "//" : "transitions test code for SIG_X",
```
Comment (sombeam) [inline testing/tests/moduletests/SIG/SIG_01.json]: test code **setup** ?

File: testing/tests/moduletests/SIG/SIG_02.cpp [hunk 6]
```diff
@@ -12,10 +12,6 @@ namespace test
     {
     protected:
         SIG_Hub mSUT;
-        inline TestGroupSIH()
-        {
-            mockAll();
```
Comment (sombeam) [inline testing/tests/moduletests/SIG/SIG_02.cpp:17]: check for mockAll in all module tests

File: testing/tests/moduletests/TSW/README.md [hunk 7]
```diff
@@ -6,6 +6,11 @@ test of module TSW
 according to test plan TSW_states.md
```
Comment (sombeam) [inline testing/tests/moduletests/TSW/README.md]: remove all test plan comments

File: testing/tests/moduletests/TSW/TSW_01.cpp [hunk 8]
```diff
@@ -45,112 +45,107 @@ namespace test
     TEST_GROUP_BASE(TSW_01, TestGroupTSW) {};
 
     //  test type: equivalence class test
-    //  all TSW transitions acc. to TSW_states.md
```
Comment (sombeam) [inline testing/tests/moduletests/TSW/TSW_01.cpp:48]: all transitions
generated transitions

**Review by sombeam — CHANGES_REQUESTED** [review]

File: CLOC.md (hunk 1 above)
Comment (sorgom) [reply CLOC.md]: done

File: application/components/SIG/SIG_X.h (hunk 2 above)
Comment (sorgom) [reply application/components/SIG/SIG_X.h]: done

File: testing/plans/tests_LCR_X.csv
Comment (sorgom) [reply testing/plans/tests_LCR_X.csv:1]: removed plans folder

File: testing/tests/moduletests/LCR/LCR_01.json (hunk 3 above)
Comment (sorgom) [reply testing/tests/moduletests/LCR/LCR_01.json]: ok, changed for TSW, LCR, SIG

File: testing/tests/moduletests/SIG/SIG_01.json (hunk 5 above)
Comment (sorgom) [reply testing/tests/moduletests/SIG/SIG_01.json]: done

File: testing/tests/moduletests/SIG/README.md (hunk 4 above)
Comment (sorgom) [reply testing/tests/moduletests/SIG/README.md]: done

File: testing/tests/moduletests/SIG/SIG_02.cpp (hunk 6 above)
Comment (sorgom) [reply testing/tests/moduletests/SIG/SIG_02.cpp:17]: ok

File: testing/tests/moduletests/TSW/README.md (hunk 7 above)
Comment (sorgom) [reply testing/tests/moduletests/TSW/README.md]: done

File: testing/tests/moduletests/TSW/TSW_01.cpp (hunk 8 above)
Comment (sorgom) [reply testing/tests/moduletests/TSW/TSW_01.cpp:48]: done

**Review by sombeam — APPROVED** [review]

**Pull request merged**